chapter_01/p04_palindrome_permutation.py

```diff
@@ -36,7 +36,8 @@ def char_number(c):
 
 def is_palindrome_permutation_pythonic(phrase):
     """function checks if a string is a permutation of a palindrome or not"""
-    counter = Counter(phrase.replace(" ", "").lower())
+    chars = set([chr(97 + i) for i in range(26)])
+    counter = Counter([c.lower() for c in phrase if c.lower() in chars])
```

Comment:
lets use `string.ascii_lowercase` instead:
```suggestion
    counter = Counter([c for c in phrase.lower() if c in string.ascii_lowercase])
```

Reply:
Thank you for that! `string.ascii_lowercase` is very convenient syntax to know :)